Meadowlark-js/packages/meadowlark-core/src/handler/CrudHandler.ts

```diff
@@ -6,16 +6,6 @@
 /* eslint-disable-next-line import/no-unresolved */
```

Comment:
Some of the functions in this module are very lengthy. Personally, I usually prefer < 15 lines in a function. Looking at the details, I'm not sure that's realistic. I think there's also a bunch of refactoring we can do when we get to the point of teasing apart the front end and the business logic or service layer.

Just mentioning this for discussion.

Reply:
Agree, they are longer than I'd like. As you say, I think some of this will end up spilt between front end and meadowlark-core.